demisto_sdk/commands/doc_reviewer/rn_checker.py

```diff
@@ -44,9 +58,15 @@ class ReleaseNotesChecker:
         'You can now',
         'Deprecated. ',
```

Comment:
```suggestion
        'Deprecated.',
```
Maybe move this template to the suffix template set